creator-node/test/nodesync.test.js

```diff
@@ -667,115 +667,129 @@ describe('test nodesync', async function () {
         .get(uri => uri.includes('/export'))
         .reply(200, sampleExport)
 
+      // This text 'audius is cool' is mapped to the hash in the dummy json data
+      // If changes are made to the response body, make the corresponding changes to the hash too
+      nock('http://mock-cn1.audius.co')
+        .persist()
+        .get(uri => uri.includes('/ipfs'))
+        .reply(200, 'audius is cool')
+
+      nock('http://mock-cn2.audius.co')
+        .persist()
+        .get(uri => uri.includes('/ipfs'))
+        .reply(200, 'audius is cool')
+
+      nock('http://mock-cn3.audius.co')
+        .persist()
+        .get(uri => uri.includes('/ipfs'))
+        .reply(200, 'audius is cool')
```

Comment:
was thinking we should have all these mocks be if the uri includes the expected hash, so `uri.includes('/ipfs/QmSU6rdPHdTrVohDSfhVCBiobTMr6a3NvPz4J7nLWVDvmE')`, nbd since this works but prob better